scripts/performance-tests-kevm.sh

```diff
@@ -106,16 +106,30 @@ feature_shell "cd kevm-pyk && poetry run pytest src/tests/integration/test_prove
 
 mkdir -p $SCRIPT_DIR/logs
 
+# use special options if given, but restore KORE_RPC_OPTS afterwards
+if [ ! -z "${FEATURE_SERVER_OPTS}" ]; then
+    echo "Using special options '${FEATURE_SERVER_OPTS}' via KORE_RPC_OPTS"
+    if [ ! -z "${KORE_RPC_OPTS:-}" ]; then
+        PRIOR_OPTS=${KORE_RPC_OPTS}
+    fi
+    export KORE_RPC_OPTS=${FEATURE_SERVER_OPTS}
+fi
```

Comment:
I put this in ad-hoc, it is helpful when new optional features are introduced (here: running with `--simplify-each`) but those options won't show in the log files for the timing, unfortunately.